README.md

```diff
@@ -1,8 +1,761 @@
+
+<!-- <br/>
+</blockquote> -->
+</details>
+
+<details>
+<summary>Integrate with a Bitbucket repo and deploy pull requests</summary>
+<!-- <blockquote>
+<br/> -->
+
+1. Create a free trial:
+
+   [Register for a 30 day free trial with Platform.sh](https://auth.api.platform.sh/register). When you have completed signup, select the **Create from scratch** project option. Give you project a name, and select a region where you would like it to be deployed. As for the *Production environment* option, make sure to match it to this repository's settings, or to what you have updated the default branch to locally.
+
+1. Install the Platform.sh CLI
+
+   #### Linux/OSX
+
+   ```bash
+   curl -sS https://platform.sh/cli/installer | php
+   ```
+
+   #### Windows
+
+   ```bash
+   curl -f https://platform.sh/cli/installer -o cli-installer.php
+   php cli-installer.php
+   ```
+
+   You can verify the installation by logging in (`platformsh login`) and listing your projects (`platform project:list`).
+
+1. Create the repository
+
+   Create a new repository on Bitbucket, set it as a new remote for your local copy, and push to the default branch.
+
+1. Setup the integration:
+
+   Consult the [Bitbucket integration documentation](https://docs.platform.sh/integrations/source/bitbucket.html#setup) to finish connecting a repository to a project on Platform.sh. You will need to create an Access token on Bitbucket to do so.
+
+<!-- <br/>
+</blockquote> -->
+</details>
+
+### Migrating your data
+
+If you are moving an existing site to Platform.sh, then in addition to code you also need to migrate your data. That means your database and your files.
+
+<details>
+<summary>Importing the database</summary><br/>
+
+First, obtain a database dump from your current site and save your dump file as `database.sql`. Then, import the database into your Platform.sh site using the CLI:
+
+```bash
+platform sql -e main < database.sql
+```
+
+</details>
+<details>
+<summary>Importing files</summary><br/>
+
+You first need to download your files from your current hosting environment.
+The easiest way is likely with rsync, but consult your old host's documentation.
+
+The `platform mount:upload` command provides a straightforward way to upload an entire directory to your site at once to a `mount` defined in a `.platform.app.yaml` file.
+Under the hood, it uses an SSH tunnel and rsync, so it is as efficient as possible.
+(There is also a `platform mount:download` command you can use to download files later.)
+Run the following from your local Git repository root (modifying the `--source` path if needed and setting `BRANCH_NAME` to the branch you are using).
+
+A few examples are listed below, but repeat for all directories that contain data you would like to migrate.
+
+```bash
+$ platform mount:upload -e main --mount web/sites/default/files --source ./web/sites/default/files
```

Comment:
Looks like `var` may be the only dir that needs uploading? https://github.com/symfonycorp/platformsh-symfony-template/blob/bb433871d2dda2d29b815776a0af43f5ffaef7ea/.platform.app.yaml#L36

Reply:
yes